test/CMakeLists.txt

```diff
@@ -1 +1,2 @@
+add_subdirectory(common)
```

Comment:
I moved the catch.hpp and the main file into a the common directory as I'm planning to add some integration-type tests later.

Reply:
yeah, it takes too long to build catch's main